src/main/java/seedu/address/logic/commands/DeleteCommand.java

```diff
@@ -20,29 +21,45 @@ public class DeleteCommand extends Command {
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
             + ": Deletes the person identified by the index number used in the displayed person list.\n"
-            + "Parameters: INDEX (must be a positive integer)\n"
-            + "Example: " + COMMAND_WORD + " 1";
+            + "Parameters: INDICES (must be a positive integer or a range, separated by commas)\n"
+            + "Example: " + COMMAND_WORD + " 1 2 3 5-7";
```

Comment:
Message says "separated by commas", but indices are separated by whitespace in the example